core/refinerycms-core.gemspec

```diff
@@ -23,7 +23,6 @@ Gem::Specification.new do |s|
   s.required_ruby_version = Refinery::Version.required_ruby_version
 
   s.add_dependency 'refinerycms-i18n',            ['~> 4.0', '>= 4.0.0']
-  s.add_dependency 'awesome_nested_set',          ['~> 3.0', '>= 3.0.0']
```

Comment:
nice, I guess core doesn't use this. Hopefully if extensions use it they specify it.